WebRoot/skins/_base/base4/skin.properties

```diff
@@ -921,8 +921,8 @@ ListItemBgColor-hover           = @lighten(AppC,25)@
 ListItemBgColor-active          = @lighten(AppC,25)@
 ListItemBgColor-matched         = @MatchedColor@
 ListItemBgColor-selected        = @AltC@
-ListItemBgColor-sel-alt         = @SelAltColor@
-ListItemBgColor-actioned        = @SelC@
+ListItemBgColor-sel-alt         = @AltC@
```

Comment:
This change will have one side effect. Please check `Create Contact Group` page and also `Preference => Signature Page`

Thanks

Reply:
Fixed in https://github.com/Zimbra/zm-web-client/pull/163/commits/4839250d72ed00dd9a882eb6499c282f7f5a896d